.prettierignore

```diff
@@ -35,5 +35,5 @@ spec.emu.html
 docs/**/reference/
 packages/website/versioned_docs/**/reference/
 
-# Cadl init template
+#.tsp init template
```

Comment:
leading '.'  (in comment, so not urgent)

Reply:
Should this be "TypeSpec" ?